src/mainviews/create-job.tsx

```diff
@@ -101,10 +105,26 @@ export function CreateJob(props: ICreateJobProps): JSX.Element {
           newComputeType = envList[0].default_compute_type;
         }
 
+        const outputFormats = outputFormatsForEnvironment(
+          envList,
+          envList[0].name
+        )?.map(format => format.name);
+
         props.handleModelChange({
           ...props.model,
           environment: envList[0].name,
-          computeType: newComputeType
+          computeType: newComputeType,
+          outputFormats: outputFormats
+        });
+        environment = envList[0].name;
+      } else {
+        const outputFormats = outputFormatsForEnvironment(
+          envList,
+          environment
+        )?.map(format => format.name);
```

Comment:
You don't need to set `outputFormats` here in the `else` case since `props.model.environment` is an empty string. We don't select a default environment in the case where there is more than one environment supported.

Reply:
This makes sense, thanks for explanations